src/core_tests.rs

```diff
@@ -398,28 +396,30 @@ fn random_churn<R: Rng>(rng: &mut R,
 }
 
 // Get names of all nodes in the `bucket_index`-th bucket in the routing table.
-fn actual_names_in_bucket(table: &RoutingTable<XorName>, bucket_index: usize) -> BTreeSet<XorName> {
-    let our_name = table.our_name();
-    let far_name = our_name.with_flipped_bit(bucket_index);
-
-    table.closest_nodes_to(&far_name, GROUP_SIZE, false)
-        .into_iter()
-        .map(|info| *info.name())
-        .filter(|name| our_name.bucket_index(name) == bucket_index)
-        .collect()
-}
+// fn actual_names_in_bucket(table: &RoutingTable<XorName>,
+//                           bucket_index: usize)
+//                           -> BTreeSet<XorName> {
+//     let our_name = table.our_name();
+//     let far_name = our_name.with_flipped_bit(bucket_index);
+
+//     table.closest_nodes_to(&far_name, MIN_GROUP_SIZE, false)
+//         .into_iter()
+//         .map(|info| *info.name())
+//         .filter(|name| our_name.bucket_index(name) == bucket_index)
+//         .collect()
+// }
```

Comment:
Do we use these at all ? If not I would remove them, otherwise they should be completed. Is the intention to do that in a separate PR or remove them altogether